conda/common/anaconda_client.py

```diff
@@ -145,19 +162,13 @@ def read_binstar_tokens():
 
 
 def set_binstar_token(url, token):
```

Comment:
TODO @srossross I need this code and `remove_binstar_token`, but only to help effectively test this module.  These two functions are *only* used in tests, and I wouldn't mind moving them to the test_anaconda_client.py module in the future.